react-swipe-to-reveal-options.js

```diff
@@ -269,7 +271,7 @@
           }
           return React.createElement("div", { className: "stro-button stro-left-button " + option.class,
             key: 'swipe-left-option-' + index,
-            onClick: this.leftClick.bind(this, option),
+            onClick: index ? this.rightClick.bind(this, option) : this.leftClick.bind(this, option),
```

Comment:
I don't understand this change, this is a button on the left, why would it be a `rightClick`?

Reply:
Oh oops. I misunderstood the api. `onRightClick` was firing for both buttons in right reveal menu. I misinterpreted that left click was for the left button and right for right button. I see now that the button clicked is passed back in options.

Closing this PR and reopening just for the onClose add